mathjax3-ts/a11y/explorer.ts

```diff
@@ -166,26 +170,29 @@ export function ExplorerMathItemMixin<B extends Constructor<HTMLMATHITEM>>(
         public updateDocument(document: ExplorerMathDocument) {
             super.updateDocument(document);
             this.refocus && this.typesetRoot.focus();
-            this.restart && this.explorers.forEach(x => x.Start());
+            this.restart && this.attached.forEach(x => x.Start());
             this.refocus = this.restart = false;
         }
 
-    };
-
-}
+        /**
+         * Adds a list of explorers and makes sure the right one stops propagating.
+         * @param {Explorer[]} explorers The active explorers to be added.
+         */
+        private addExplorers(explorers: Explorer[]) {
+            if (explorers.length <= 1) return;
+            let lastKeyExplorer = null;
+            for (let explorer of this.attached) {
+                if (!(explorer instanceof ke.AbstractKeyExplorer)) continue;
+                if (lastKeyExplorer) {
+                    lastKeyExplorer.stoppable = false;
+                }
+                lastKeyExplorer = explorer;
+                lastKeyExplorer.stoppable = true;
+            }
+        }
```

Comment:
Would this make a little more sense as

```javascript

                    let lastKeyExplorer = null;
	            for (let explorer of this.attached) {
	                if (!(explorer instanceof ke.AbstractKeyExplorer)) continue;
	                explorer.stoppable = false;   // I'm not sure if this is needed; remove if not
	                lastKeyExplorer = explorer;
	            }
	            if (lastKeyExplorer) {
	                lastKeyExplorer.stoppable = true;
	            }
```

Reply:
`explorer.stoppable = false;` is necessary as the flag could be set previously, before another explorer was added.